### PR TITLE
Talos - Improve PR title

### LINKED ISSUE #1754: Talos - Improve PR title
**Is your feature request related to a problem? Please describe.**
The current Talos PR title is just `Talos - Bump <list of packages>` which can get quite long with alot of packages

image

**Describe the solution you'd like**
Make the PR name, `Talos - Bump <package1> <package2> <package3>...` Where you only show a max of 3 packages in the list

**Additional context**

### PR DESCRIPTION
**Pull request by FK78**
Resolves #1754

**Overall change:** _Shortened the PR titles_

**Code changes:**

- _Created new function called getPackageNames._
- _Created new file with unit tests for new function._
- _Added unit test to createPullRequest for new function._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: scripts/talos/createPullRequest/getPackageNames/index.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import getPackageNames from '.';
+
+describe('getPackageNames', () => {
+  it('should shorten the title', () => {
```
Comment (dr3) [inline scripts/talos/createPullRequest/getPackageNames/index.test.js]:
```suggestion
  it('should shorten the title when more than 3 packages are provided', () => {
```

File: scripts/talos/createPullRequest/getPackageNames/index.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    expect(
+      getPackageNames(['package1', 'package2', 'package3', 'package4']),
+    ).toEqual('package1, package2, package3...');
+  });
+
+  it('should not shorten the title', () => {
```
Comment (dr3) [inline scripts/talos/createPullRequest/getPackageNames/index.test.js]:
```suggestion
  it('should not shorten the title when less than 3 packages are provided', () => {
```

**Review by dr3 — CHANGES_REQUESTED** [review]
LGTM, just 2 small things :)

**Review by dr3 — APPROVED** [review]

**Review by chris-hinds — APPROVED** [review]

**Review by thekp — APPROVED** [review]

**Pull request merged**